src/pages/trainer/training/edit/actions/fetchTrainingContent.ts

```diff
@@ -0,0 +1,19 @@
+import {trainerActionEnums} from '../../../../../common/actionEnums/trainer';
+import {trainerApi} from '../../../../../rest-api';
+
+export const fetchTrainingContentStarted = (trainingId: number) => {
+  return (dispatcher) => {
+    const promise = trainerApi.getTrainingConentByTrainingId(trainingId);
+
+    promise.then(
+      (data) => dispatcher(fetchTrainingContentCompleted(data)),
+    );
+
+    return promise;
+  };
```

Comment:
Sorry, can you refresh my mind, it is because redux-thunk, that you have to return the promise?